README.md

```diff
@@ -1 +1,51 @@
-# IdolGAN
\ No newline at end of file
+# IdolGAN ~~[adoːkẽꜜɴ]~~
+
+We use [HiFaceGAN](https://github.com/Lotayou/Face-Renovation) and [KID-F Dataset](https://github.com/PCEO-AI-CLUB/KID-F).
+
+# Usage
+### Environment
+- Ubuntu/CentOS
+- PyTorch 1.0+
+- CUDA 10.1
+- python packages: opencv-python, tqdm, 
+- Data augmentation tool: [imgaug](https://imgaug.readthedocs.io/en/latest/source/installation.html#installation-in-pip)
+- [Face Recognition Toolkit](https://github.com/ageitgey/face_recognition) for evaluation
+- [tqdm](https://github.com/tqdm/tqdm) to make you less anxious when testing:)
+
```

Comment:
Also, we miss our mediapipe for face detection.
Now we just need requirements for crop.py & degrad.py. 
Like tqdm, python, mediapipe, cv2, etc. Pls check that.